CRM/Donrec/Form/Task/ContributeTask.php

```diff
@@ -30,12 +30,21 @@ function buildQuickForm() {
        'last_year'         => E::ts('last year'),
        'customized_period' => E::ts('Choose Date Range')
     );
-    $this->addElement('select', 'time_period', 'Time Period:', $options, array('class' => 'crm-select2'));
-    $this->addDateRange('donrec_contribution_horizon', '_from', '_to', E::ts('From:'), 'searchDate', TRUE, FALSE);
+    $this->addDatePickerRange(
+      'donrec_contribution_horizon',
+      E::ts('Time period'),
+      FALSE,
+      FALSE,
```

Comment:
As pointed out in the PR summary, setting this to `TRUE` causes all sub-fields be required, which makes selecting relative date options impossible.